TECHMANIA/Assets/Scripts/Components/Main Menu/OptionsPanel.cs

```diff
@@ -311,15 +311,15 @@ public void OnRulesetChanged()
             }
             catch (System.IO.FileNotFoundException)
             {
-                messageDialog.Show(Locale.GetStringAndFormat(
+                messageDialog.Show(Locale.GetStringAndFormatIncludingPaths(
```

Comment:
Sorry I didn't catch this earlier, but `HidePlatformInternalPath` doesn't cover the custom ruleset file.